### PR TITLE
Fix wasm ci failure due to dns

### PR DESCRIPTION
**Pull request by itsankitkp**

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -473,6 +473,7 @@ jobs:
       - uses: Swatinem/rust-cache@v2
       - name: Setup Wasmer
         uses: wasmerio/setup-wasmer@v2
+        security: strict
```
Comment (fanninpm) [inline .github/workflows/ci.yaml:476]:
```suggestion
        with:
          security: strict
```

Comment (itsankitkp) [reply .github/workflows/ci.yaml]: Fixed

**Comment by itsankitkp** [comment]
> I'd recommend opening an issue upstream to https://github.com/wasmerio/setup-wasmer.

Yes, issue lies on their side

**Comment by itsankitkp** [comment]
Closing this PR as it is not something we can fix here

**Pull request closed without merging**